core_lib_generator/file_generators/core_lib_class_generator.py

```diff
@@ -14,12 +15,17 @@ def generate(self, template_content: str, yaml_data: dict, core_lib_name: str, f
             updated_file = _add_job(updated_file, yaml_data, core_lib_name)
         if yaml_data['config']:
             updated_file = _add_mongo(updated_file, yaml_data['config']['data'], core_lib_name)
+        if yaml_data['entities']:
+            updated_file = _add_alembic_funcs(updated_file, yaml_data['entities'], core_lib_name)
         return updated_file
 
     def get_template_file(self, yaml_data: dict) -> str:
         return 'core_lib_generator/template_core_lib/template_core_lib/template_core_lib.py'
 
 
+tab_spaces_8 = 8

```

Comment:
this is that character in a single tab? if its 2 tabs then call 
`add_tab_spaces(str, 2)` (see suggestion below)